app/src/organisms/ErrorRecoveryFlows/ErrorRecoveryWizard.tsx

```diff
@@ -83,20 +83,29 @@ export function ErrorRecoveryWizard(
 export function ErrorRecoveryComponent(
   props: RecoveryContentProps
 ): JSX.Element {
-  const { recoveryMap, hasLaunchedRecovery, isDoorOpen, isOnDevice } = props
+  const { recoveryMap, isDoorOpen, isOnDevice } = props
   const { route, step } = recoveryMap
   const { t } = useTranslation('error_recovery')
   const { showModal, toggleModal } = useErrorDetailsModal()
 
   const buildTitleHeading = (): JSX.Element => {
-    const titleText = hasLaunchedRecovery ? t('recovery_mode') : t('cancel_run')
-    return <LegacyStyledText as="h4Bold">{titleText}</LegacyStyledText>
+    const titleText = props.hasLaunchedRecovery
```

Comment:
```suggestion
    const titleText =hasLaunchedRecovery
```
and destructure `hasLaunchedRecovery` above.